src/protocol/memcache/get.rs

```diff
@@ -113,15 +124,17 @@ async fn run_get(
             BACKEND_EX.increment();
 
             klog_1(&"get", &key, Status::ServerError, 0);
-            None
+            Err(Error::new(ErrorKind::Other, format!("{e}")))
         }
         Err(_) => {
+            error!("backend timeout for get");
```

Comment:
This log does not provide any extra information compared to the metric and can be removed.